### PR TITLE
feat: Add new job of running unit tests in MacOS for cicd

### PR DESCRIPTION
**Pull request by tobegit3hub**
Resolve https://github.com/4paradigm/SparkFE/issues/37 .

### CODE REVIEW

**Comment by tobegit3hub** [comment]
Refer to https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md , maven 3.8.1 is used in MacOS runner.

Refer to https://maven.apache.org/docs/3.8.1/release-notes.html , maven 3.8.1 disable HTTP repository.

File: .github/workflows/cicd.yml [hunk 1]
```diff
@@ -21,18 +21,40 @@ on:
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+      - name: Compile for all-in-one
+        run: |
+          cd sparkfe/ # TODO: Run the unit tests if dynamic library works
```
Comment (imotai) [inline .github/workflows/cicd.yml:55]: @jingchen2222 What is the status of library for macos and can we use it on macos ?

Comment (tobegit3hub) [reply .github/workflows/cicd.yml:55]: Now we can only build the SparkFE in MacOS but can not run the unit tests. If the environment has been setup, we can run the tests but not in this pr now.

**Comment by dl239** [comment]
LGTM

**Review by dl239 — APPROVED** [review]

**Pull request merged**